torch/csrc/jit/codegen/cuda/python_frontend/fusion_record.h

```diff
@@ -1510,6 +1511,95 @@ struct BatchNormOpRecord : RecordFunctor {
   bool channels_last_;
 };
 
+struct FullOpRecord : RecordFunctor {
+  FullOpRecord(
+      std::vector<State> _args,
+      std::vector<State> _outputs,
+      std::vector<int64_t>& shape,
+      Nvf::DataType dtype)
+      : RecordFunctor(
+            std::move(_args),
+            std::move(_outputs),
+            "full",
```

Comment:
If you make this a more generic op, the name would be passed through. The names need to have the prefix `ops` in order to be reproducible.  Like `ops.full`.